projects/plugins/jetpack/extensions/blocks/ai-assistant/transforms/index.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { createBlock } from '@wordpress/blocks';
+/**
+ * Internal dependencies
+ */
+import { blockName } from '..';
+import { EXTENDED_BLOCKS, isPossibleToExtendBlock } from '../extensions/ai-assistant';
+import { PromptItemProps } from '../lib/prompt';
+
+const transformFromCoreEmbed = {
+	type: 'block',
+	blocks: EXTENDED_BLOCKS,
+	isMatch: () => isPossibleToExtendBlock(),
+	transform: ( { content } ) => {
+		const messages: Array< PromptItemProps > = [
+			{
+				role: 'user',
```

Comment:
```suggestion
				role: 'assistant',
```

I didn't test the code yet, but I guess it would make sense to use the current content from the paragraph as the last response from the assistant, as if it got generated from the assistant before, so the follow up actions would work based on it.

Going to test it now to see if the follow ups are working as expected.

Reply:
Tested it and so far it's working as expected using the `role: user` for the existing content 💯 so I think we are cool.

Reply:
Also tested using `role: assistant` locally and it works as well. Maybe some edge case will show the better approach to us.

Reply:
`assistant` looks better to me, at least for now.